### PR TITLE
pinning GRB parameters

### PR DESCRIPTION
**Pull request by sahiljhawar**
pinning GRB parameters to avoid empty light curves error

### CODE REVIEW

File: nmma/tests/injections.py [hunk 1]
```diff
@@ -59,6 +59,8 @@ def create_injection_from_command_line(model_name):
 
         if model_name == "nugent-hyper":
             prior_path = os.path.join( priorDir, "sncosmo-generic" + ".prior")
+        elif model_name == "TrPi2018":
+            prior_path = os.path.join( dataDir, "constTrPi2018" + ".prior") #pinning the parameter svalues in the prior file
```
Comment (mcoughlin) [inline nmma/tests/injections.py:63]: Can we call this TrPi2018_pinned_parameters or similar?

**Review by mcoughlin — CHANGES_REQUESTED** [review]
This is good, should just change the parameter name.

File: nmma/tests/injections.py (hunk 1 above)
Comment (sahiljhawar) [reply nmma/tests/injections.py]: Done

Comment (mcoughlin) [reply nmma/tests/injections.py]: @sahiljhawar TrPi2018_pinned_parameters.prior, you are missing the period.

Comment (sahiljhawar) [reply nmma/tests/injections.py]: Sorry, my bad!

**Review by mcoughlin — APPROVED** [review]
LGTM

**Pull request merged**